packages/utils/src/assetData/baseAssets.ts

```diff
@@ -136,6 +139,21 @@ export const tcy: Readonly<Asset> = Object.freeze({
   relatedAssetKey: null,
 })
 
+export const maya: Readonly<Asset> = Object.freeze({
+  assetId: mayaTokenAssetId,
+  chainId: caip.mayachainChainId,
+  name: 'MAYA',
+  networkName: 'MAYAChain',
+  symbol: 'MAYA',
+  precision: 4,
+  color: '#00FA0',
```

Comment:
```suggestion
  color: '#00FFA0',
```

Reply:
ser I don't see any commit with this diff, must be a bug